### PR TITLE
Replace remaining `querySubgraph` and `querySubgraphExhaustive` instances

### PR DESCRIPTION
**Pull request by peripheralist**
- Add a bunch of new .graphql queries based on existing file pattern, and replace remaining `querySubgraph` and `querySubgraphExhaustive` instances
- Updates to `paginateDepleteQuery` for apollo queries, which replaces `useSubgraphExhaustive`

### CODE REVIEW

File: codegen.yml [hunk 1]
```diff
@@ -17,4 +17,7 @@ config:
   namingConvention:
     enumValues: keep
   scalars:
-    BigInt: BigNumber
\ No newline at end of file
+    BigInt: BigNumber
+  avoidOptionals:
+    field: true
+  skipTypename: true
```
Comment (peripheralist) [inline codegen.yml:23]: - `avoidOptionals` prevents typing some things as `undefined` that in practice will only ever be defined or `null`
- `skipTypename` avoids expecting a `typename: string` property in Apollo responses, which creates some type compatibility issues

File: src/lib/apollo/paginateDepleteProjectsQuery.ts [hunk 2]
```diff
@@ -15,25 +17,9 @@ export async function paginateDepleteProjectsQueryCall({
 }: {
   variables: QueryProjectsArgs
 }) {
-  const first = variables.first ?? 1000
-  const skip = variables.skip
-
-  const { data } = await serverClient.query<ProjectsQuery, QueryProjectsArgs>({
-    query: ProjectsDocument,
-    variables: { ...variables, first, skip },
+  return paginateDepleteQuery<ProjectsQuery, QueryProjectsArgs>({
```
Comment (peripheralist) [inline src/lib/apollo/paginateDepleteProjectsQuery.ts:20]: this is just a wrapper for the frequently-used projects-`paginateDepleteQuery` that makes reuse cleaner

File: src/lib/apollo/paginateDepleteQuery.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import { DocumentNode } from 'graphql'
+
+/**
+ * Loads all pages for the query.
+ * @param client ApolloClient to use for query.
+ * @param document GraphQL docuemnt to define query.
+ * @param variables Variables to apply to query.
+ * @returns
+ */
+export async function paginateDepleteQuery<
```
Comment (peripheralist) [inline src/lib/apollo/paginateDepleteQuery.ts:12]: new generic paginateDepleteQuery that replaces `useSubgraphExhaustive`

File: src/utils/csvDownloadHelpers.ts [hunk 4]
```diff
@@ -1,8 +1,35 @@
...
+} from 'generated/graphql'
+import { client } from 'lib/apollo/client'
+import { paginateDepleteQuery } from 'lib/apollo/paginateDepleteQuery'
 import { PV } from 'models/pv'
 import { downloadCsvFile } from 'utils/csv'
 import { fromWad } from 'utils/format/formatNumber'
-import { querySubgraphExhaustive } from 'utils/graph'
 import { emitErrorNotification } from 'utils/notifications'
 
 export async function downloadParticipants(
```
Comment (peripheralist) [inline src/utils/csvDownloadHelpers.ts:35]: just replacing a lot of subgraph queries with apollo queries

File: src/utils/csvDownloadHelpers.ts [hunk 5]
```diff
@@ -295,7 +281,7 @@ export async function downloadV1Payouts(
       rows.push([
         date,
         fromWad(p.modCut),
-        fromWad(p.modCutUSD),
+        p.modCutUSD ? fromWad(p.modCutUSD) : 'n/a',
```
Comment (peripheralist) [inline src/utils/csvDownloadHelpers.ts:284]: although it shouldn't ever be in practice, `amountUSD` is nullable in the subgraph and correctly typed via Apollo, so we handle that type case here

File: codegen.yml (hunk 1 above)
Comment (tomquirk) [reply codegen.yml:23]: ay

**Review by tomquirk — APPROVED** [review]

**Pull request merged**